src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php

```diff
@@ -69,6 +69,10 @@
         'Symfony\Component\DependencyInjection\ContainerInterface' => 'Symfony\Component\VarDumper\Caster\StubCaster::cutInternals',
         'Symfony\Component\VarDumper\Exception\ThrowingCasterException' => 'Symfony\Component\VarDumper\Caster\ExceptionCaster::castThrowingCasterException',
 
+        'PHPUnit_Framework_MockObject_MockObject' => 'Symfony\Component\VarDumper\Caster\StubCaster::cutInternals',
+        'PHPUnit_Framework_MockObject_Stub' => 'Symfony\Component\VarDumper\Caster\StubCaster::cutInternals',
+        'Prophecy\Prophecy\ProphecyInterface' => 'Symfony\Component\VarDumper\Caster\StubCaster::cutInternals',
```

Comment:
I think this should be applied on ``ProphecySubjectInterface``, not on ``ProphecyInterface``. ``Prophecy\Prophecy\ProphecyInterface`` is the object used to configure the mock. If you are dumping this one, you probably want to see the internals. The test double will implement ``Prophecy\Prophecy\ProphecySubjectInterface``